### ISSUE TITLE
Malaria Cell Image Classification

### ISSUE THREAD

**Issue by sitamgithub-MSIT** [abhisheks008/DL-Simplified#393]
### Deep Learning Simplified Repository (Proposing a New Issue)
:red_circle: **Project Title** : Malaria Cell Image Classification
:red_circle: **Aim**: The malaria dataset contains cell images with equal instances of parasitized and uninfected cells from the thin blood smear slide images of segmented cells. The task is to perform image classification on that.
:red_circle: **Dataset** : https://www.tensorflow.org/datasets/catalog/malaria
:red_circle: **Approach** : Try to use 3–4 algorithms to implement the models and compare all the algorithms to find the best-fit algorithm for the model by checking the accuracy scores. Also, do not forget to do exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project:**
* You need to create a separate folder named Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or information or source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages and libraries to run the project on other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Sitam Meur
- GitHub Profile Link : https://github.com/sitamgithub-MSIT
- Email ID : sitammeur@gmail.com
- Participant ID (if applicable):
- Approach for this Project:
1. Import necessary libraries like Keras and Keras-cv
2. Performing the necessary pre-processing, image resizing, rescaling, and all.
3. Creating all the required functions. Probably some visualization utility functions and others.
4. Split the data into test and training datasets.
5. Train the model on VGG-16, Resnet-50, and other algorithms if needed.
6. Checking the validation accuracy and loss.
7. At the end, label all the accuracy of all models in a table.
- What is your participant role? (Mention the Open Source program.) Codepeak'23 Contributor.

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open-source journey ahead. 😎

**Comment by sitamgithub-MSIT** [comment]
@abhisheks008 Would you please review this issue? and designate it for me.

**Comment by abhisheks008** [comment]
Issue assigned to you @sitamgithub-MSIT
